pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[tool.poetry]
+name = "pygui"
+version = "0.1.0"
+description = "Reactive rendering for Python"
+authors = ["Korijn van Golen <korijn@gmail.com>", "Berend Klein Haneveld <berendkleinhaneveld@gmail.com>"]
+license = "MIT"
+homepage = "https://github.com/fork-tongue/pygui"
+readme = "README.md"
+
+[tool.poetry.extras]
+gfx = ["pygfx"]
+pyside = ["PySide6"]
+
+[tool.poetry.dependencies]
+python = ">=3.8"
+# NOTE: When observ is updated, the python marker can be removed
+observ = { version = "~0.9.0", python = "<3.11" }
+pygfx = { git = "https://github.com/pygfx/pygfx.git", rev = "main", optional = true }
+PySide6 = { version = "*", python = "<3.11", optional = true }
+
+[tool.poetry.dev-dependencies]
+black = "*"
+flake8 = "*"
+flake8-black = "*"
+flake8-import-order = "*"
+flake8-print = "*"
+pre-commit = "*"
+pytest = "*"
+pytest-cov = "*"
+rich = { version = "*", optional = true, python = "<4.0" }
```

Comment:
How does optional work in dev dependencies? :o

Reply:
Ehhh, great question, I don't know :D I'll just remove it